src/app/lib/config/services/ws.ts

```diff
@@ -200,6 +200,7 @@ export const service: DefaultServiceConfig = {
         video: 'Video',
         listen: 'Listen',
         watch: 'Watch',
+        watchMoments: 'Watch Moments',
```

Comment:
Do we need to include this translation?

Reply:
I re-added this as I thought it was the cause of some errors, but this can be removed. it's not needed

Reply:
Cool, thanks for sorting that.